wcivf/apps/elections/templates/elections/includes/_polling_place.html

```diff
@@ -3,116 +3,124 @@
-          {% else %}
-            your local council.</p>
-          {% endif %}
+            {% if postcode|ni_postcode %}
+                <p>You will need photographic identification.</p>
+            {% endif %}
+        {% endif %}
+        <p>
+            {% if postcode|ni_postcode %}
+                <a href="http://www.eoni.org.uk/Vote/Voting-at-a-polling-place" target="_top">
+                    Read more about how to vote in Northern Ireland</a>.
+            {% else %}
+                <a href="https://www.gov.uk/voting-in-the-uk/polling-stations" target="_top">
+                    Read more about how to vote</a>.
+            {% endif %}
+        </p>
+
+        {% if polling_station.polling_station_known and polling_station.polling_station.geometry %}
+            <p>
+                <a href="https://wheredoivote.co.uk/postcode/{{ postcode }}">
+                    Get walking directions from {{ postcode }}</a>.
+            </p>
+        {% endif %}
+
+        {% if not polling_station.custom_finder and polling_station.polling_station_known and polling_station.polling_station.geometry %}
+            <div id="area_map" style="height:400px;"></div>
+
+            <link rel="stylesheet" href="//cdnjs.cloudflare.com/ajax/libs/leaflet/0.7.7/leaflet.css" />
+            <script src="//cdnjs.cloudflare.com/ajax/libs/leaflet/0.7.7/leaflet.js"></script>
+
+            <script type="text/javascript">
+                // Maps
+                window.create_area_map = function(polling_station_point) {
+                    var polling_station_location = polling_station_point;
+                    window.polling_station_location = polling_station_location;
+
+                    var map = L.map('area_map', {
+                        zoomControl: true
+                    });
+                    map.dragging.disable();
+                    // map.touchZoom.disable();
+                    // map.doubleClickZoom.disable();
+                    map.scrollWheelZoom.disable();
+
+                    L.tileLayer('https://{s}.tile.openstreetmap.org/{z}/{x}/{y}.{ext}', {
+                        ext: 'png',
+                        attribution: 'Map data © <a href="https://openstreetmap.org">OpenStreetMap</a> contributors',
+                        subdomains: 'abc'
+                    }).addTo(map);
+
+                    window.map = map;
+
+                    L.marker(polling_station_location, {
+                        'clickable': true,
+                    }).addTo(map);
+
+                    map.setView(polling_station_location, 15);
+                };
+
+                var polling_station_point = [{
+                    {
+                        polling_station.polling_station.geometry.coordinates .1
+                    }
+                },
+                    {
+                        {
+                            polling_station.polling_station.geometry.coordinates .0
+                        }
+                    }
```

Comment:
Looks like these have been broken from formatting too, they won't be interpreted as the Django template variable correctly so should go back to:
```
var polling_station_point = [{
    {{ polling_station.polling_station.geometry.coordinates.1 }},
    {{ polling_station.polling_station.geometry.coordinates.0 }},
}]
```